src/crypto/local.js

```diff
@@ -373,6 +394,43 @@ function getSeedPRNG(chkM, chkZ, pointG, pointH) {
     return sjcl.codec.hex.fromBits(h.finalize());
 }
 
+/**
+ * hkdf - The HMAC-based Key Derivation Function
+ * based on https://github.com/mozilla/node-hkdf
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
I think it would be good to mention that this doesn't handle a case when ``len(salt)==0``
From https://tools.ietf.org/html/rfc5869
```

      salt     optional salt value (a non-secret random value);
               if not provided, it is set to a string of HashLen zeros.
```

Reply:
This is handled, see KATs